grade.go

```diff
@@ -366,7 +365,26 @@ func main() {
 		return
 	}
 
-	if isDryRun {
+	assignmentInfo := util.ParseAssignmentOgInfo(workDir + "/.spec/oginfo.json")
+
+	if language == "" {
+		language = assignmentInfo.Language
+	}
+
+	if runArgs == "" {
+		runArgs = assignmentInfo.Args
+	}
+
+	if !wall {
```

Comment:
this will fire if wall is provided and false... which might actually be fine

we did have it default to true though iirc